scripts/derivechangeset

```diff
@@ -0,0 +1,7 @@
+
+# HootApiDb requires a map layer, though (hence, 'blah').
+step1:
+	hoot derive-changeset $(HOOT_OPTS) -D convert.bounding.box=$(aoi) -D osm.changeset.sql.writer.generate.new.ids=false $(input1) $(input2) $(output) $(jobid) $(DB_URL)/blah
+
```

Comment:
From what I saw in the services, full url's are never passed in by the ui to the services.  The ui only passes in layer names and then hoot always prepends the hoot api db url to the layer name before importing/conflating/exporting, etc.  I see what you're saying though, in that workflow since one of the layers would always end up being hootapidb:// (we're assuming no .osm file inputs), then the url prefix before the layer name could be grabbed from the input instead to eliminate the extra param, I guess.  But if the whole thing gets changed to how export works with temp file, then I guess its moot anyway.